tests/utils/test_testing.py

```diff
@@ -73,15 +70,12 @@ def test_assert_elements_are_identical_metadata(full_sdata):
     for _, element_name, element in to_iter:
         if get_model(element) in (Image2DModel, Image3DModel):
             _change_metadata_image(copied, element_name, coords=True, transformations=False)
-            # TODO: bug in data_tree, need to report
-            if not isinstance(copied[element_name], MultiscaleSpatialImage):
-                with pytest.raises(AssertionError):
-                    assert_elements_are_identical(full_sdata[element_name], copied[element_name])
+            if not isinstance(copied[element_name], DataTree):
+                assert_elements_are_identical(full_sdata[element_name], copied[element_name])
             _change_metadata_image(copied, element_name, coords=True, transformations=True)
         elif get_model(element) in (Labels2DModel, Labels3DModel):
             _change_metadata_labels(copied, element_name)
-            # TODO: bug in data_tree, need to report
-            if not isinstance(copied[element_name], MultiscaleSpatialImage):
+            if not isinstance(copied[element_name], DataTree):
                 with pytest.raises(AssertionError):
                     assert_elements_are_identical(full_sdata[element_name], copied[element_name])
```

Comment:
you removed one of the two `with.pytest.raises()`, I would bet that either both are needed or both are not. If both are needed is to silence a bug with copying that I had found in `data_tree` (I actually don't remember what the bug was, I should have written it in the comment). If you end up keeping the `with.pytest.raises()` please restore the comment to explain that the objects should be identical but a bug in `data_tree` makes the comparison fail.